src/app/components/category-card/category-card.component.html

```diff
@@ -0,0 +1,95 @@
+<div class="card">
+  <div class="loading-indicator" *ngIf="loading === true">
+    <mat-progress-spinner
+      mode="indeterminate"
+      color="accent"
+    ></mat-progress-spinner>
+  </div>
+  <form [formGroup]="categoryForm" (ngSubmit)="onSubmit()">
+    <div class="action-container">
+      <button
+        class="button-category"
+        (click)="onEditClick()"
+        *ngIf="editMode === false"
+      >
+        <mat-icon>edit</mat-icon>
+      </button>
+      <button
+        (click)="onDelete()"
+        *ngIf="editMode === false"
+        class="button-category"
+        [disabled]="canMakeChanges ? false : true"
+      >
+        <mat-icon>delete</mat-icon>
+      </button>
+      <button
+        class="button-category"
+        *ngIf="editMode === true"
+        type="submit"
+      >
+        <mat-icon>save</mat-icon>
+      </button>
+      <button
+        class="button-category"
+        *ngIf="editMode === true"
+        (click)="onDiscard()"
+      >
+        <mat-icon>close</mat-icon>
+      </button>
+    </div>
+    <div class="segment flex-center">
+      <mat-form-field appearance="fill">
+        <mat-label> Title. </mat-label>
+        <input
+          matInput
+          [readonly]="canMakeChanges && editMode ? null : true"
+          formControlName="titleControl"
+          placeholder="Choose category name."
+        />
+        <mat-error
+          *ngIf="
+            categoryForm.get('titleControl')!.hasError('required') &&
+            categoryForm.touched
+          "
+        >Title length is incorrect.</mat-error>
+      </mat-form-field>
+    </div>
+
+    <div class="segment flex-center">
+      <div class="centered-segment flex-center">
+        <app-upload-file
+          [form]="categoryForm"
+          [hidden]="editMode ? false : true"
```

Comment:
It is enough to write [hidden]="!editMode".
This way, when editMode is true hidden will be false and if editMode is false, hidden will be true